### ISSUE TITLE
Bulk update support

### ISSUE THREAD

**Issue by lorefnon** [juanluispaz/ts-sql-query#52]
Hello, hope you are doing well.

Is it possible to construct a bulk update query that does not need a temporary table (supported by postgres) : 

```sql
   UPDATE REGISTRATION AS t
   SET name = c.column_name, modified = c.column_date_modified
   FROM (
      VALUES  (1, 'test one', '12–12–2019'),
      (2, 'test two', '12–12–2019'),
      (3, 'test three','12–12–2019'),
      (4, 'test four', '12–12–2019')
   ) AS c(column_id, column_name, column_date_modified)
   WHERE c.column_id = t.id
```

The values construct is described [here](https://www.postgresql.org/docs/current/sql-values.html).

**Comment by juanluispaz** [comment]
Hi,

I will implement it (the values construction you showed me in your example) , just let me figure out a convenient way to do it

**Comment by juanluispaz** [comment]
Implemented in ts-sql-query 1.41.0 already published. Implement this feature was challenging because corner cases, specially in PostgreSQL.

Some documentation:

- [Bulk update](https://ts-sql-query.readthedocs.io/en/stable/queries/update/#bulk-update)
- [Bulk delete](https://ts-sql-query.readthedocs.io/en/stable/queries/delete/#bulk-delete)
- [Mapping constant values as view](https://ts-sql-query.readthedocs.io/en/stable/connection-tables-views/#mapping-constant-values-as-view)
- [Constant values view definition](https://ts-sql-query.readthedocs.io/en/stable/supported-operations/#constant-values-view-definition)

Can you review it and let me know any feedback?

**Comment by lorefnon** [comment]
This is great, thanks a lot. This will simplify a bunch of places where we are using raw sql now.
